src/preproc/preprocessing_pass_registry.cpp

```diff
@@ -0,0 +1,52 @@
+/*********************                                                        */
+/*! \file preprocessing_pass_registry.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **  Justin Xu
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Preprocessing pass registry
+ **
+ ** Preprocessing pass registry that registers the preprocessing passes.
```

Comment:
This description could talk about implementation details, e.g. that passes are stored in a map.